src/nvim/buffer_defs.h

```diff
@@ -649,7 +649,7 @@ struct file_buffer {
   bool b_p_swf;                 /* 'swapfile' */
   long b_p_smc;                 /* 'synmaxcol' */
   char_u      *b_p_syn;         /* 'syntax' */
```

Comment:
Nope. Review needs to review *all* changes. It is also OK if the only commit that passes linting is the last one.